arvo/ames/ames.md

```diff
@@ -355,23 +381,28 @@ private key and the public key of `~worwel-sipnum`, obtained via Jael. Then
 The packet has the following format:
  - The standard Ames header described in [header](#header), where the checksum
    is the `+mug` of the body, and the sender and receiver ship types are `01`,
-   denoting that the sender and receiver are planets. The remainder of the
-   packet is the body.
- - The body of this packet will be the `@p` of the
-sender `~bacbel-tagfeb`, followed by the `@p` of the receiver `~worwel-sipnum`,
-followed by the payload.
- - The payload of this packet will be the `+jam` of `[origin content]`. `origin`
-for this initial packet will be `~`, which implies that the packet is
-originating from the sender. `content` is the `+jam` of the message, encrypted
-using the Diffie-Hellman symmetric key that was previously computed.
+   denoting that the sender and receiver are planets. The "is this relayed?" bit
+   is set to 0 since this is the first hop on the packet route.
+ - The body of this packet will be the life of `~bacbel-tagfeg` mod 16, followed
+   by the life of `~worwel-sipnum` mod 16, followed by `~bacbel-tagfeg`,
+   followed by the receiver `~worwel-sipnum`, followed by the origin `~`
+   (denoting that the ship sending the packet is the origin). After this comes
+   the payload.
+ - The payload of this packet will be the `+jam` of the `content`, which is an
+ encrypted fragment of the message `%watch /path/to/recipes`.
 
 `~zod` receives the packet and reads the body. It sees that it is not the
 intended recipient of the packet, and so gets ready to forward it to
-`~worwel-sipnum`. First, it [`+cue`s](@/docs/hoon/reference/stdlib/2p.md#cue)
-(deserializes) the payload and changes the `origin` to the IP and port of
-`~bacbel-tagfeb`. Then it `+jam`s `[origin content]` to form a new payload, and
-`+mug`s that payload to get a new checksum. It replaces the old payload and
-checksum with the new, and then sends the packet along to `~worwel-sipnum`.
+`~worwel-sipnum`. First, it replaces the origin field with the IP and port of
+`~bacbel-tagfeb`, then computes the `+mug` of the body and replaces the checksum
+with the new mug. Since the packet is being relayed, `~zod` flips the "is it
+relayed?" bit to 1. `~zod` then forwards the packet to `~worwel-sipnum`.
```

Comment:
The 'relayed' bit is a loobean, so `relayed` is `&`, or `%.y`, which is 0, not 1.